### PR TITLE
New recipe for Babel

### PR DESCRIPTION
**Pull request by frol**
[Babel](http://babel.pocoo.org) is an integrated collection of utilities that assist in internationalizing and localizing Python applications, with an emphasis on web-based applications.

### CODE REVIEW

File: recipes/babel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - babel
+  commands:
+    - pybabel --help
+
+about:
+  home: http://babel.pocoo.org/
+  license: BSD License
```
Comment (ocefpaf) [inline recipes/babel/meta.yaml]: What type 2 or 3 clause?

File: recipes/babel/bld.bat [hunk 2]
```diff
@@ -0,0 +1,2 @@
+%PYTHON% setup.py install
+if errorlevel 1 exit 1
```
Comment (ocefpaf) [inline recipes/babel/bld.bat]: Same as the other :wink:

**Comment by frol** [comment]
I have addressed the comments.

**Comment by ocefpaf** [comment]
Thanks @frol!

**Pull request merged**